sdk/appconfiguration/app-configuration/package.json

```diff
@@ -48,12 +48,12 @@
     "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
     "build:test:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
     "build:samples": "dev-tool samples prep && cd dist-samples && tsc",
-    "check-format": "prettier --list-different --config ../../.prettierrc.json --ignore-path ../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
```

Comment:
This issue existed before this PR, but I wonder if hardcoding the path to the `.prettierrc.json` is the best way to share config.  Would it instead be better to publish a module that exports a config object, as recommended at https://prettier.io/docs/en/configuration.html#sharing-configurations?  This would also allow individual packages to override the config if needed.

Reply:
I also wonder if setting the prettier config this way might enable more features (like IDE support), since we are setting the config the "correct way" rather than just hardcoding a path to `.prettierrc.json` in the `check-format` and `format` scripts.

Reply:
`prettier` resolves the configuration file starting from the location of the file being formatted, and searching up the file tree until a config file is (or isn’t) found. So we do not really need to hard code paths anyway as long as it can be resolved by `prettier`. I do not feel strongly about either approaches since `prettier` already works in vscode without problems using the current setup, so I am happy with either options.

Reply:
I wasn't aware `prettier` searched up the file tree for config files.  I agree this is simpler than publishing a module, that approach would be more appropriate if config needed to be shared across repos.

If the `prettier` commands in `package.json` work correctly without the `--config` and `--ignore-path` parameters, then I think they should be removed in this PR.

Reply:
For some reason, `prettier` is not able to locate the config files without `--config` and `--ignore-path` parameters in `format` and `check-format` scripts. I opened #10422 to investigate this further.